test-assets/detect-changes-configs/ntp-release/jobs/ntpd/spec~

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I have two aliases for convenience:
The first simply prints all under CWD. The second removes them.
 ```
alias ftilde=`find    . -name "*~" -print'
alias rtilde=`find    . -name "*~" -exec rm "{}" \;'
```